nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/rest/RestESDocumentsTest.java

```diff
@@ -128,7 +128,7 @@ public void iCanPerformESQLPageProviderOnRepository() throws IOException, Interr
 
         JsonNode node = mapper.readTree(response.getEntityInputStream());
         // Verify results
-        assertEquals(19, getLogEntries(node).size());
+        assertEquals(20, getLogEntries(node).size());
```

Comment:
So are we confident we know what happens here? It's worth getting to the bottom of this to be sure we don't have a hidden bug somewhere else.